docs/roman/stpipe/index.rst

```diff
@@ -13,7 +13,6 @@ For Users
    user_pipeline.rst
    user_logging.rst
    config_asdf.rst
-   call_via_call.rst
```

Comment:
We might need to restore this file too, for the reasons pointed by @braingram regarding the use of `run` and `call`.

Reply:
I think that is out of scope here and the call vs run rewrite is in a separate ticket

Reply:
Would you elaborate on this? Specifically:
- What's the separate ticket?
- Is romancal planning to not use `Step.call`? That's news to me.